### PR TITLE
Lighting: The cleanup.

### PR DESCRIPTION
**Pull request by PJB3005**
Cleans up light source code, less copy pasta (including @Mloc's define).

Moves luminosity changes to the overlays instead of the turfs.

Lowers some light power values since we're not running lambertian.

### CODE REVIEW

File: code/modules/lighting/light_source.dm [hunk 1]
```diff
@@ -126,76 +126,89 @@
...
+    #define LUM_ATTENUATION(swapvar) swapvar = CLAMP01(swapvar / max(1,light_range))
+  #endif
+#endif
+
+#define LUM_FALLOFF(swapvar, O, T) \
+  LUM_DISTANCE(swapvar, O, T); \
+  LUM_ATTENUATION(swapvar);
+
+#define APPLY_TURF(T)						\
+	if(T.lighting_overlay)					\
```
Comment (N3X15) [inline code/modules/lighting/light_source.dm]: jesus H christ

you do realize that this ends up just copypasting the code wherever you use APPLY_TURF(T) right

Comment (PsiOmegaDelta) [inline code/modules/lighting/light_source.dm]: This is true, but at the same time the code is defined in this singular place.

Comment (PJB3005) [inline code/modules/lighting/light_source.dm]: @N3X15 Yes, that's exactly what I'm going for.

**Comment by clusterfack** [comment]
You set luminosity to 0 in some areas of centcomm like thunderdome

**Comment by PJB3005** [comment]
Ah fuck I realised what caused it.

**Comment by Zth--** [comment]
And the cargo shuttle

File: code/modules/power/lighting.dm [hunk 2]
```diff
@@ -110,7 +110,7 @@ var/global/list/obj/machinery/light/alllights = list()
 	var/on_gs = 0
 	var/static_power_used = 0
 	var/brightness_range = 8	// luminosity when on, also used in power calculation
-	var/brightness_power = 4
+	var/brightness_power = 1
```
Comment (Probe1) [inline code/modules/power/lighting.dm]: CANT SEE SHIT CAPTAIN

File: code/modules/power/lighting.dm [hunk 3]
```diff
@@ -159,7 +159,7 @@ var/global/list/obj/machinery/light/alllights = list()
 	fitting = "large tube"
 	light_type = /obj/item/weapon/light/tube/large
 	brightness_range = 8
-	brightness_power = 3
+	brightness_power = 1
```
Comment (Probe1) [inline code/modules/power/lighting.dm]: Its a flood light tho

**Comment by PJB3005** [comment]
@Probe1 The shading we're using right now (linear) should be the same as with DAL, so at 1 power this should be right.

**Comment by Probe1** [comment]
Should be but now the station is significantly more dim.  I'm about to add like 12 lights to Box

**Pull request merged**